sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -536,10 +536,12 @@ pi_result piContextCreate(const pi_context_properties *properties,
 pi_result piextContextCreateWithNativeHandle(pi_native_handle nativeHandle,
                                              pi_uint32 num_devices,
                                              const pi_device *devices,
+                                             bool ownNativeHandle,
                                              pi_context *piContext) {
   (void)num_devices;
   (void)devices;
   assert(piContext != nullptr);
+  assert(ownNativeHandle == false);
```

Comment:
This assert looks redundant. Probably it was added by mistake.

Reply:
Why do you think it is redundant? It checks that OpenCL plugin isn't asked to "own" the handle, because it doesn't support it.

Reply:
Oh, I see, thx, I thought that this is a Level Zero plugin by mistake.